CMakeLists.txt

```diff
@@ -733,14 +733,15 @@ if (${PLAYER_BUILD_EXECUTABLE})
 
 	target_link_libraries(${EXE_NAME} ${PROJECT_NAME})
 
-	# installation - don't use install on macs
-	if(NOT APPLE)
-		include(GNUInstallDirs)
+	include(GNUInstallDirs)
+	if(APPLE)
+		install(TARGETS ${EXE_NAME} RUNTIME DESTINATION BUNDLE DESTINATION "${CMAKE_BINARY_DIR}/Package")
```

Comment:
Is the use of CMAKE_BINARY_DIR intentional here? That will install in the build-tree.

Reply:
Yes. Where else to put it? `/usr/local` is not inappropriate, and Jenkins wants to call install...